src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMRevision.java

```diff
@@ -81,6 +88,16 @@ public String getPullHash() {
         return pullHash;
     }
 
+    /**
+     * The commit hash of the head of the pull request branch.
+     *
+     * @return The commit hash of the head of the pull request branch
+     */
+    @CheckForNull
```

Comment:
It _will_ be null if you load an old serialized object, and yet the one caller is not checking for null.

Reply:
This is once again, POC.  The scenario for the caller is guaranteed to never have null.